Encodings.md

```diff
@@ -247,6 +253,15 @@ and handled as wrapping around in 2's complement notation so that the original
 values are correctly restituted. This may require explicit care in some programming
 languages (for example by doing all arithmetic in the unsigned domain).
 
+One strategy that might be employed to avoid the above mentioned overflow is to
+perform the subtraction utilizing integers with a larger number of bits. For example,
+while encoding INT32 data one might choose to perform arithmetic operations using
+64-bit integers. This can lead to situtations where the number of bits used to encode
+the resulting deltas is greater than the number of bits used to represent the input
+values. While this behavior is allowed, data produced in this manner may not be
```

Comment:
I don't think that this behavior is (or should be) allowed. The spec should IMHO prescribe that INT32 is encoded at most using 32-bit deltas, and INT64 using 64-bit deltas. Emitting deltas larger than the physical bitwidth should be considered a bug in the encoder.

Reply:
Actually maybe some legacy encoder generate these kind of data?

Reply:
Perhaps, but I think we should still consider it a bug :-)